forge-gui/res/cardsfolder/upcoming/waterlogged_teachings_inundated_archive.txt

```diff
@@ -2,8 +2,6 @@ Name:Waterlogged Teachings
 ManaCost:3 UB
 Types:Instant
 A:SP$ ChangeZone | ChangeType$ Instant,Card.hasKeywordFlash | ChangeNum$ 1 | Origin$ Library | Destination$ Hand | Shuffle$ True | SpellDescription$ Search your library for an instant card or a card with flash, reveal it, put it into your hand, then shuffle.
-#TODO (copied from Mystical Teachings): The AI will currently search for the most expensive valid card in the library. This can be used to the AI's advantage with
```

Comment:
Don't think we put comments like this anymore (this is better as a github issue ?)

Reply:
indeed